.github/workflows/spell-checker.yml

```diff
@@ -0,0 +1,32 @@
+name: Spell Checkerr
+on:
+  push:
+    branches:
+      - 'master'
+      - '[0-9]+.[1-9][0-9]*.x'
+    paths:
+      - 'docs/**'
+  pull_request:
+    branches:
+      - 'master'
+      - '[0-9]+.[1-9][0-9]*.x'
+    paths:
+      - 'docs/**'
+
+jobs:
+  run:
+    name: Spell Check with Typos
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Actions Repository
+      uses: actions/checkout@v2
+
+    - name: Check spelling of all files
+      uses: crate-ci/typos@master
+      with: 
+        files: ./docs
+
+    - name: Writes changes in the local checkout
```

Comment:
i'm curious to see if this is possible with our setup of getting contributions from external forks, let's see. Maybe we need to remove this.

Reply:
yes ,it will give output log whenever there is an error just like this 
![image](https://github.com/keptn/lifecycle-toolkit/assets/81870866/7a4aa56e-f814-4ba6-97d0-c90fca1bcdce)